### PR TITLE
Refine java doc

### PR DESCRIPTION
**Pull request by jiaweiteo**

### CODE REVIEW

File: src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/delete/DeleteTravelPlanCommand.java [hunk 1]
```diff
@@ -11,6 +11,9 @@
 import seedu.address.model.travelplan.TravelPlan;
 import seedu.address.model.travelplanner.Model;
 
+/**
+ * Deletes a travel plan in the travel planner identified using the index from the travel planner list.
```
Comment (lyeyixian) [inline src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/delete/DeleteTravelPlanCommand.java:15]: The Ui didn't show the index actually, do you think we wanna show it? Then, need add into Ui part

File: src/main/resources/view/TravelPlannerPanel.fxml [hunk 2]
```diff
@@ -8,7 +8,7 @@
 <?import javafx.scene.control.ListCell?>
 <VBox fx:id="travelPlanPanel" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
     <padding>
-        <Insets top="5" right="5" bottom="5" left="5" />
+        <Insets top="5" right="5" bottom="5" left="5"  />
```
Comment (lyeyixian) [inline src/main/resources/view/TravelPlannerPanel.fxml:11]: !?

**Review by lyeyixian — APPROVED** [review]
LGTM

File: src/main/resources/view/TravelPlannerPanel.fxml (hunk 2 above)
Comment (jiaweiteo) [reply src/main/resources/view/TravelPlannerPanel.fxml:11]: ??? i got changed wtf

File: src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/delete/DeleteTravelPlanCommand.java (hunk 1 above)
Comment (jiaweiteo) [reply src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/delete/DeleteTravelPlanCommand.java:15]: ok

Comment (jiaweiteo) [reply src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/delete/DeleteTravelPlanCommand.java:15]: i think we should show index in ui

**Pull request merged**